clients/da-vinci-client/src/main/java/com/linkedin/davinci/blobtransfer/NettyP2PBlobTransferManager.java

```diff
@@ -185,7 +186,7 @@ private void processPeersSequentially(
     chainOfPeersFuture.thenRun(() -> {
       if (!resultFuture.isDone()) {
         resultFuture.completeExceptionally(
-            new VenicePeersNotFoundException(String.format(NO_VALID_PEERS_MSG_FORMAT, replicaId)));
+            new VenicePeersAllFailedException(String.format(NO_VALID_PEERS_MSG_FORMAT, replicaId)));
```

Comment:
If it is no peer found, then should we mask this exception in the below check instead?